tsconfig.lib.json

```diff
@@ -2,20 +2,22 @@
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
     "jsx": "react-jsx",
+    "emitDeclarationOnly": true,
     "declaration": true,
-    "declarationDir": "dist/dts",
+    "declarationDir": "dts",
     "baseUrl": "src/lib",
     "paths": {
       "lib/*": ["./*"],
       "abis/*": ["../abis/*"],
       "assets/*": ["../assets/*"],
       "constants/*": ["../constants/*"],
       "hooks/*": ["../hooks/*"],
+      "locales/*": ["../locales/*"],
       "state/*": ["../state/*"],
       "types/*": ["../types/*"],
-      "utils/*": ["../utils/*"],
-    },
+      "utils/*": ["../utils/*"]
+    }
   },
-  "exclude": ["node_modules", "src/lib/**/*.test.*"],
-  "include": ["src/lib/**/*"]
+  "exclude": ["node_modules", "src/**/*.test.*"],
+  "include": ["src/lib/**/*.d.ts", "src/lib/index.tsx"]
```

Comment:
is this redundant with `emitDeclarationOnly`?

Reply:
No, this is a declaration file that must be included so that asset imports function (for CRA compatibility).
`src/lib/index.tsx` is not required, as it's included by the rollup toolchain, but including it allows you to run `tsc` as a standalone, so it's here for completeness.